src/Sidesheet/index.js

```diff
@@ -103,19 +104,21 @@ const Sidesheet = ({
             <div
               ref={portal}
               style={{
-                transition: "transform .2s cubic-bezier(.3,0,0,1)",
-                transform: transition
-                  ? `translateX(0)`
-                  : "translateX(100%)",
-                top: 0,
-                bottom: 0,
-                right:0,
                 width: width,
                 maxWidth: "calc(100vw - 20px)",
                 height: "calc(100% - 20px)",
               }}
-              tw="fixed z-50 min-w-0 bg-white duration-300 delay-200 h-full flex flex-col shadow-xl m-2 rounded"
-            >
+              css={[
+                css`
+                  transform: translateX(${ transition ? '0' : '100%' })
+                `,
+                css`
+                @media (prefers-reduced-motion) {
+                  transform: none;
+                  opacity: ${ transition ? '1' : '0' }
+                }`,
```

Comment:
Maybe we could try defining animations in the tailwind config https://tailwindcss.com/docs/animation#customizing-your-theme?